### ISSUE TITLE
scroll to current day when Today is clicked

### ISSUE THREAD

**Issue by kyledecot** [fullcalendar/fullcalendar-scheduler#159]
When `defaultView` is set to `timelineYear` the "Today" button will simply go to the current year. It will not however scroll to the current date.

http://jsbin.com/kizenohobe/edit?js,output

**Comment by bjornoss** [comment]
Hi. I have the same problem with timelineMonth and defaultDate: "2016-05-30" (late in the month). 
The date specified by defaultDate should be visible when the calendar loads. Thanks

**Comment by okliv** [comment]
a workaround for those who need it now

onclick event:

`$('.fc-scroller').scrollLeft($('th[data-date="'+moment().format("YYYY-MM-DD")+'"]').last().offset().left-screen.width/2);`

**Comment by syssi** [comment]
Can you suggest a good event for the initial pre-scroll workaround? I have three switchable views. The event viewRender fires too early and the eventAfterAllRender does just fire once.

**Comment by arshaw** [comment]
you might want to do `viewRender` with a `setTimeout` in it, as a hack

**Comment by syssi** [comment]
I feared that. Thanks!

**Comment by tmiveda** [comment]
Is there a chance this (accepted) feature request manages it in any next version?

**Comment by laoneo** [comment]
If somebody else is stumbling across this issue, I made a plain JS function to center the day when something changes. Hope this will get fixed soon.

```
var adaptScroll = function () {
	var cell = calendar[0].querySelector('th[data-date^="' + calendar.fullCalendar('getDate').format("YYYY-MM-DD") + '"]');
	if (!cell) {
		return;
	}

	var scroller = calendar[0].querySelector('.fc-time-area .fc-scroller');

	if (!scroller) {
		return;
	}

	scroller.scrollLeft = 0;

	var scrollerBounds = scroller.getBoundingClientRect();
	var cellBounds = cell.getBoundingClientRect();

	scroller.scrollLeft = cellBounds.left + cellBounds.width / 2 - scrollerBounds.left - scrollerBounds.width / 2;
}
```

**Comment by laoneo** [comment]
Why did you close?

**Comment by kyledecot** [comment]
@laoneo I closed it as it's been open for 3 years w/ no resolution and I don't use this library anymore. I have reopened it for others that may still be having this issue.

**Comment by AnthonySB** [comment]
Yes I could do with a default option in the scheduler to scroll the scheduler to today's date by default when the control is initially loaded.
